TSG.md

```diff
@@ -105,6 +105,23 @@ For HNS account, always add `type: adls` under `azstorage` section in your confi
 
 To create a private-endpoint for DFS in Azure portal: Go to your storage account -> Networking -> Private Endpoint connections. Click `+ Private endpoint`, fill in Subscription, Resource Group, Name, Network Interface Name and Region. Click next and under Target sub-resource select `dfs`. Click Virtual network and select virtual network and Subnet. Click DNS. Select Yes for Integrate with private DNS. Select the Subscription and Resource Group for your private link DNS. Select Next, Next and select Create.
 
+**11. Failed to initialize new pipeline [config error in azstorage [account name not provided]]'**
+
+Make sure the configuration file has `azstorage` section in your config file.
+
+```
+azstorage:
+  type: adls
+  account-name: <name of the storage account>
+  container: <name of the storage container to be mounted>
+  endpoint: <storage account endpoint (example - https://account-name.blob.core.windows.net)>
+  mode: key
+  account-key: <storage account key>
+```
```

Comment:
You can remove this section. The sample config files linked below already mentions this.